### PR TITLE
MKL-DNN 1.1 for Windows

### PR DESCRIPTION
**Pull request by grygielski**
Test PR for MKL-DNN 1.1 build for Windows

### CODE REVIEW

**Comment by grygielski** [comment]
@zhouwei25 @luotao1 
This PR passed Windows_CI_Inference (http://ci.paddlepaddle.org/viewLog.html?buildId=262184&). Please check if solution is acceptable and ask questions if any.

**Comment by zhwesky2010** [comment]
> @zhouwei25 @luotao1
> This PR passed Windows_CI_Inference (http://ci.paddlepaddle.org/viewLog.html?buildId=262184&). Please check if solution is acceptable and ask questions if any.

It's good!  I'll double check tomorrow.

File: cmake/inference_lib.cmake [hunk 1]
```diff
@@ -86,7 +86,7 @@ function(copy_part_of_thrid_party TARGET DST)
         set(dst_dir "${DST}/third_party/install/mkldnn")
         if(WIN32)
             copy(${TARGET}
-                    SRCS ${MKLDNN_INC_DIR} ${MKLDNN_SHARED_LIB} ${MKLDNN_LIB}
+                    SRCS ${MKLDNN_INC_DIR} ${MKLDNN_SHARED_LIB}  ${MKLDNN_LIB}
```
Comment (zhouwei25) [inline cmake/inference_lib.cmake]: `SRCS ${MKLDNN_INC_DIR} ${MKLDNN_SHARED_LIB} ${MKLDNN_SHARED_LIB_1} ${MKLDNN_LIB} ${MKLDNN_LIB_OLD}`
Should there be a total of five here?

**Review by luotao1 — COMMENTED** [review]
I don't see the usage of `${MKLDNN_LIB_OLD}`

**Comment by zhwesky2010** [comment]
This method of generating lib is proved to be correct. LGTM

**Comment by zhwesky2010** [comment]
> MKLDNN_LIB

It use ${MKLDNN_LIB} instead of ${MKLDNN_LIB_OLD}

**Review by luotao1 — APPROVED** [review]

**Pull request merged**